packages/slice-machine/server/src/api/tracking/onboarding.ts

```diff
@@ -0,0 +1,34 @@
+import { getEnv } from "@lib/env";
+import { FakeResponse } from "@lib/models/common/http/FakeClient";
+import {
+  OnboardingTrackingEvent,
+  TrackingEventId,
+} from "@lib/models/common/TrackingEvent";
+
+export class HTTPResponseError extends Error {
+  response: Response | FakeResponse;
+  constructor(response: Response | FakeResponse) {
+    super(`HTTP Error Response: ${response.status} ${response.statusText}`);
+    this.response = response;
+  }
+}
+
+export default async function (
+  query: Pick<
```

Comment:
Didn't know the Type Pick before, it's interesting.
Although if every Key can be optional, I would use `Partial<T>`

I'm wondering, why would the query object be `"Picked"` or `Partial` ?
The information are sent by the Client, you know the object structure so i'm wondering what's that about.

Reply:
it's been removed